udf/normalize_main_payload.sql

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
I'd recommend enforcing formatting on this udf, since it looks likely that this will be modified in the future.

Reply:
I will run the formatter on it. Do we have a way now of marking certain files for enforcing formatting?

Reply:
adding a file to this list will enforce formatting: https://github.com/mozilla/bigquery-etl/blob/master/.circleci/config.yml#L52-L60

Reply:
Done